<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FPImage</class>
 <widget class="QMainWindow" name="FPImage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1264</width>
    <height>852</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FPImage</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,5,0">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QPlainTextEdit" name="ERes"/>
      </item>
      <item>
       <widget class="QLineEdit" name="EFile"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="BLoad">
          <property name="text">
           <string>Load</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="BDoIt">
          <property name="text">
           <string>Do it</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>809</width>
         <height>777</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="Ecran">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="LRDesvMulti">
        <property name="text">
         <string>L desviation multiplier</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="SliderRDesv">
        <property name="maximum">
         <number>200</number>
        </property>
        <property name="sliderPosition">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="LGDesvMulti">
        <property name="text">
         <string>S desviation multiplier</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="SliderGDesv">
        <property name="maximum">
         <number>200</number>
        </property>
        <property name="sliderPosition">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="LBDesvMulti">
        <property name="text">
         <string>B desviation multiplier</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="SliderBDesv">
        <property name="maximum">
         <number>200</number>
        </property>
        <property name="sliderPosition">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="BDetectSkin">
        <property name="text">
         <string>Detect Skin</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="LMaskKernelX">
        <property name="text">
         <string>Mask kernel X size</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="SliderBlobKernelX">
        <property name="minimum">
         <number>3</number>
        </property>
        <property name="maximum">
         <number>15</number>
        </property>
        <property name="singleStep">
         <number>2</number>
        </property>
        <property name="pageStep">
         <number>4</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="LMaskKernelY">
        <property name="text">
         <string>Mask kernel Y size</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="SliderBlobKernelY">
        <property name="minimum">
         <number>3</number>
        </property>
        <property name="maximum">
         <number>15</number>
        </property>
        <property name="singleStep">
         <number>2</number>
        </property>
        <property name="pageStep">
         <number>4</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="BDilate">
        <property name="text">
         <string>Dilate</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="BErode">
        <property name="text">
         <string>Erode</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="minimumSize">
         <size>
          <width>256</width>
          <height>500</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="tab_3">
         <attribute name="title">
          <string>Histograms Brightness and Contrast</string>
         </attribute>
         <widget class="QLabel" name="EcranHistoR">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>260</width>
            <height>104</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>260</width>
            <height>104</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>260</width>
            <height>104</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="lineWidth">
           <number>2</number>
          </property>
          <property name="midLineWidth">
           <number>0</number>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLabel" name="EcranHistoG">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>100</y>
            <width>260</width>
            <height>104</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>260</width>
            <height>104</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>260</width>
            <height>104</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="lineWidth">
           <number>2</number>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLabel" name="EcranHistoB">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>200</y>
            <width>260</width>
            <height>104</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>260</width>
            <height>104</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>260</width>
            <height>104</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="lineWidth">
           <number>2</number>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QPushButton" name="BHistogram">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>300</y>
            <width>256</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Do Histograms</string>
          </property>
         </widget>
         <widget class="QSlider" name="SliderBrightness">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>340</y>
            <width>256</width>
            <height>22</height>
           </rect>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>50</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
         <widget class="QLabel" name="LBrightness">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>330</y>
            <width>256</width>
            <height>13</height>
           </rect>
          </property>
          <property name="text">
           <string>Brightness</string>
          </property>
         </widget>
         <widget class="QLabel" name="LContrast">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>350</y>
            <width>256</width>
            <height>13</height>
           </rect>
          </property>
          <property name="text">
           <string>Contrast</string>
          </property>
         </widget>
         <widget class="QSlider" name="SliderContrast">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>370</y>
            <width>256</width>
            <height>22</height>
           </rect>
          </property>
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>50</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
         <widget class="QLabel" name="TransferenceFunction">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>390</y>
            <width>256</width>
            <height>256</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>256</width>
            <height>256</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>256</width>
            <height>256</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="lineWidth">
           <number>2</number>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_4">
         <attribute name="title">
          <string>Tab 2</string>
         </attribute>
         <widget class="QLabel" name="LEdges">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>0</y>
            <width>256</width>
            <height>13</height>
           </rect>
          </property>
          <property name="text">
           <string>Edges</string>
          </property>
         </widget>
         <widget class="QSlider" name="SliderEdges">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>10</y>
            <width>256</width>
            <height>22</height>
           </rect>
          </property>
          <property name="maximum">
           <number>300</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
         <widget class="QGroupBox" name="RadioEdgeGroup">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>40</y>
            <width>256</width>
            <height>100</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>100</height>
           </size>
          </property>
          <property name="title">
           <string>Edge Detection Method</string>
          </property>
          <widget class="QRadioButton" name="EuclideanRButton">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>20</y>
             <width>260</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>Euclidean</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QRadioButton" name="AbsDifferenceRButton">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>40</y>
             <width>260</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>AbsDifference</string>
           </property>
          </widget>
          <widget class="QRadioButton" name="MaxDifferenceRButton">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>60</y>
             <width>260</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>MaxDifference</string>
           </property>
          </widget>
          <widget class="QCheckBox" name="CheckRandomDithering">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>80</y>
             <width>256</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>Random Dithering</string>
           </property>
          </widget>
         </widget>
         <widget class="QLabel" name="LAdaptEq">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>140</y>
            <width>256</width>
            <height>13</height>
           </rect>
          </property>
          <property name="text">
           <string>Adapatative equalization Neighborhood</string>
          </property>
         </widget>
         <widget class="QSlider" name="SliderAdaptEq">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>160</y>
            <width>256</width>
            <height>22</height>
           </rect>
          </property>
          <property name="minimum">
           <number>5</number>
          </property>
          <property name="maximum">
           <number>21</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
         <widget class="QLabel" name="LKernel">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>190</y>
            <width>71</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Blur Kernel</string>
          </property>
         </widget>
         <widget class="QSlider" name="SliderKernel">
          <property name="geometry">
           <rect>
            <x>-1</x>
            <y>211</y>
            <width>251</width>
            <height>21</height>
           </rect>
          </property>
          <property name="minimum">
           <number>3</number>
          </property>
          <property name="maximum">
           <number>9</number>
          </property>
          <property name="singleStep">
           <number>2</number>
          </property>
          <property name="pageStep">
           <number>2</number>
          </property>
          <property name="sliderPosition">
           <number>3</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
         <widget class="QPushButton" name="BBlurInside">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>240</y>
            <width>75</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>BlurInside</string>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1264</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <slots>
  <slot>ChangeBrightness(int)</slot>
 </slots>
</ui>
